CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -253,7 +253,11 @@ public void CancelComposition(IntPtr hwnd)
 
         private void OnImeEndComposition(IntPtr hwnd)
         {
-            owner.GetBrowserHost().ImeFinishComposingText(false);
+            if(languageCodeId != ImeNative.LANG_KOREAN)
```

Comment:
Please add a code comment here explaining the change

Reply:
The comments have been updated.